src/core/Akka.MultiNodeTestRunner.Shared/Sinks/TimelineLogCollectorActor.cs

```diff
@@ -87,26 +91,26 @@ public class LogMessageInfo
             public NodeInfo Node { get; }
             public string OriginalMessage { get; }
             public DateTime When { get; }
-            public LogLevel LogLevel { get; }
+            public LogLevel? LogLevel { get; }
             public string Message { get; }
 
             public LogMessageInfo(LogMessage msg)
             {
                 OriginalMessage = msg.Message;
                 Node = msg.Node;
                 When = DateTime.UtcNow;
-                LogLevel = LogLevel.InfoLevel; // In case if we could not find log level, assume that it is Info
+                LogLevel = null; //some log lines coming from nodes have no level (ex: stack traces) => live the level null and output always
```

Comment:
Yep, that's a good thing, making this more explicit :+1:

Reply:
*leave?

Reply:
ah, good catch :+1: "live" --> "leave"

Reply:
i've already fixed the typo - please look at the files (comments seem stuck to the original version)